packages/config/src/genesisConfig/types.ts

```diff
@@ -11,5 +11,10 @@ export interface IForkDigestContext {
 }
 
 export interface ICachedGenesis extends IForkDigestContext {
-  getDomain(domainType: DomainType, slot: Slot): Uint8Array;
+  /**
+   * Return the signature domain (fork version concatenated with domain type) of a message.
+   *
+   * @param fork The domain will be selected the previous or current version if provided. Otherwise, the chain config fork schedule is used.
```

Comment:
```suggestion
   * @param {stateFork} The domain will be selected the previous or current version if provided. Otherwise, the chain config fork schedule is used.
```